### PR TITLE
[core/ui] LIQ-653 Verified NFT collection

### PR DESCRIPTION
**Pull request by vuvu2259**

### CODE REVIEW

File: core/ui/src/components/Order/index.tsx [hunk 1]
```diff
@@ -50,8 +51,11 @@ export const Order: React.FC<OrderProps> = ({ order, wallet, walletConnectCompon
         />
         <div className="candy-order-info">
           {order?.nftAnimationLink?.includes('ext=mp4') && <IconPlayer className="candy-order-player-icon" />}
-          <div className="candy-order-name candy-line-limit-1">
-            {`${order?.name}${order?.edition !== 0 ? ` #${order?.edition}` : ''}`}
+          <div className="candy-order-name-container">
+            <div className="candy-order-name candy-line-limit-1">
+              {`${order?.name}${order?.edition !== 0 ? ` #${order?.edition}` : ''}`}
+            </div>
+            {order?.ververifiedNftCollection === 1 && <IconVerify />}
```
Comment (beepsaidblue) [inline core/ui/src/components/Order/index.tsx:58]: wrong spelling?

File: core/ui/src/public/OrderDetail/index.tsx [hunk 2]
```diff
@@ -119,7 +117,9 @@ export const OrderDetail: React.FC<OrderDetailProps> = ({
         <div className="candy-order-detail-left">{order && <Viewer order={order} />}</div>
         <div className="candy-order-detail-right">
           {isUserListing && <div className="candy-status-tag-inline">Your Listing</div>}
-          <div className="candy-order-detail-title">{order?.name}</div>
+          <div className="candy-order-detail-title">
+            {order?.name} {order?.ververifiedNftCollection === 1 && <IconVerify />}
```
Comment (beepsaidblue) [inline core/ui/src/public/OrderDetail/index.tsx:121]: wrong spelling?

File: core/ui/src/components/BuyModal/BuyModalDetail.tsx [hunk 3]
```diff
@@ -53,7 +54,9 @@ const BuyModalDetail: React.FC<BuyModalDetailProps> = ({
         <Viewer order={order} />
       </div>
       <div className="candy-buy-modal-container">
-        <div className="candy-title">{order?.name}</div>
+        <div className="candy-title">
+          {order?.name} {order?.ververifiedNftCollection === 1 && <IconVerify />}
```
Comment (beepsaidblue) [inline core/ui/src/components/BuyModal/BuyModalDetail.tsx:58]: wrong spelling?

File: core/ui/src/assets/IconVerify.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const IconVerify: React.FC = () => {
```
Comment (beepsaidblue) [inline core/ui/src/assets/IconVerify.tsx:3]: IconVerified instead of IconVerify

Comment (vuvu2259) [reply core/ui/src/assets/IconVerify.tsx:3]: Sure

**Comment by haVincy** [comment]
Test link - https://core-verified-nft-collection.d22tqykss69onq.amplifyapp.com/

**Comment by haVincy** [comment]
@vuvu2259 could you rebase to top of master?

**Comment by vuvu2259** [comment]
> @vuvu2259 could you rebase to top of master?

Done!

**Review by haVincy — APPROVED** [review]

**Pull request merged**